packages/sortable/src/components/SortableContext.tsx

```diff
@@ -11,6 +11,7 @@ export interface Props {
   items: (UniqueIdentifier | {id: UniqueIdentifier})[];
   strategy?: SortingStrategy;
   id?: string;
+  disabled?: boolean | { draggable?: boolean; droppable?: boolean }; }; 
```

Comment:
Passing `disabled` to `SortableContext` makes it possible to disable all items in the context